sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1604,6 +1604,15 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val ALLOW_CTAS_NON_EMPTY_PATH =
+    buildConf("spark.sql.legacy.ctas.allowNonEmptyLocation")
```

Comment:
ALLOW_CTAS_NON_EMPTY_LOCATION? should we keep consistence?

Reply:
Done